lib/matplotlib/widgets.py

```diff
@@ -404,11 +424,31 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
         self.val = valinit
         self.valinit = valinit
         if orientation == 'vertical':
-            self.poly = ax.axhspan(valmin, valinit, 0, 1, **kwargs)
-            self.hline = ax.axhline(valinit, 0, 1, color=initcolor, lw=1)
+            self.bkd = Rectangle((.25, 0), .5, 1, transform=ax.transAxes, facecolor=bkd_color)
```

Comment:
Can we have a more telling name?
```suggestion
            self.track = Rectangle((.25, 0), .5, 1, transform=ax.transAxes, facecolor=bkd_color)
```
seems to be the technical term.

Reply:
Ah that's a much better name thanks. I was struggling to find the correct term.